src/main/java/org/testeditor/fixture/commons/text/generate/UniqueIdGenerator.java

```diff
@@ -0,0 +1,73 @@
+/*******************************************************************************
+ * Copyright (c) 2012 - 2018 Signal Iduna Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/org/documents/epl-2.0/EPL-2.0.html
+ *
+ * Contributors:
+ * Signal Iduna Corporation - initial API and implementation
+ * akquinet AG
+ * itemis AG
+ *******************************************************************************/
+
+package org.testeditor.fixture.commons.text.generate;
+
+import java.io.UnsupportedEncodingException;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.util.UUID;
+
+import org.testeditor.fixture.core.FixtureException;
+
+public class UniqueIdGenerator {
+    
+    private final int characterLength = 64;
+    
+    
+    /**
+     * Generates a unique identifier as a String with 64 hexadecimal characters.
+     * @return A unique identifier as a String in the form 
+     * uid = 74cb6f38a27603017ed01b91433429f443b690b006c868c4288802d6241cd92d.
+     * @throws FixtureException 
+     */
+    public String generateUniqueId() throws FixtureException  {
+        String encoding = "UTF-8";
+        String algorythm = "SHA-256";
+        MessageDigest salt = null;
```

Comment:
Could you maybe add a brief sentence in the method's JavaDoc explaining how it does what it does? It don't quite get, for example, why this variable is named 'salt'. Or is this a common idiom? Then maybe just add a link to somewhere, if that's appropriate…?